### ISSUE TITLE
Not Storing token id in appCollection

### ISSUE THREAD

**Issue by mitiaptest** [raix/push#112]
Hello Raix,

I have added config.push.json into my codebase with only gcm credintailes as follows:
{
  "gcm": {
    "apiKey": "My api key",
    "projectNumber": int number
  },
  "production": true
}

Also created startup.js

Meteor.startup(function(){
  if (Meteor.isCordova){
      Push.debug = true
      console.log('startup event cordova');
      // window.alert = navigator.notification.alert;

     //ONLY This works
    Push.addListener('token', function(token) {
     console.log('token received: ' + JSON.stringify(token));
    });

    Push.addListener('error', function(err) {
     console.log(err);
    });

    Push.addListener('message', function(notification) {
     // Called on every message
     console.log(JSON.stringify(notification))
     alert(notification.message);
   })

   Push.addListener('alert', function(notification) {
     // Called on every message
     Dialog.alert(notification.message);
     console.log(notification.message);
   });
 }
})

Only i received token in console but not in my appCollection.

Thanks,

**Comment by mitiaptest** [comment]
@raix any help ?

**Comment by harryward** [comment]
Hey @mitiaptest where are you running

<code>Push.appCollection.find().fetch()</code>

Browser Console? Meteor Shell? 

My suggestion to you is to turn on Push debug

Put this somewhere server side
<code>Push.debug = true;</code>

That will show every token received in your server logs (as they are received), and will also show you the number of devices that you have tokens for by device when you go to send a push.

<code>Push: Sent message "test" to 3 ios apps 2 android apps</code>

You probably know this, but including just in case - to access the logs for your meteor app (hosted on .meteor test server) run this terminal command
<code>meteor logs serverName.meteor.com</code>

**Comment by mitiaptest** [comment]
Hey @harryward  I am running it on Meteor Shell.

**Comment by harryward** [comment]
@mitiaptest I believe you have to have ios and android credentials in your config.push.json file for raix:push to work properly. Try adding apn dev credentials and run your app in 'development' mode and see if it works for you.

**Comment by mitiaptest** [comment]
@harryward,Yes I have if I blindly set userId to query its works and send push notification to user whose userId I have set but not sure why appCollection showing is empty.
